README.md

```diff
@@ -106,6 +106,32 @@ opening a command prompt and typing
    messageix-dl --local_path /path/to/tutorials
    ```
 
+## Running Tutorials
+
+### Using Anaconda
```

Comment:
I'd think that the easiest way is to launch `Jupyter notebook` directly (not through Anaconda), then navigate to the tutorial folder.

Reply:
I didn't see that option on my Mac after installing Anaconda (only the navigator). So you have to launch it through that...

Reply:
Ok I think I addressed this now.